src/components/Resources/useResources.tsx

```diff
@@ -58,11 +59,22 @@ const formatSmallUSD = (value: number, locale: string): string =>
     maximumSignificantDigits: 2,
   }).format(value)
 
-export const useResources = ({ txCostsMedianUsd }): DashboardSection[] => {
+export const useResources = ({
+  txCostsMedianUsd,
+  totalBlobs,
+  avgBlobFee,
+}): DashboardSection[] => {
   const { t } = useTranslation("page-resources")
   const locale = useLocale()
   const localeForNumberFormat = getLocaleForNumberFormat(locale! as Lang)
 
+  const ethPrice = useEthPrice()
+  const avgBlobFeeUsd = formatSmallUSD(
+    // Converting value from gwei to USD
+    avgBlobFee * 1e-9 * ethPrice,
+    localeForNumberFormat
+  ).replace(/[A-Za-z]$/, "")
+
```

Comment:
I double-checked in a converter, and this value should be correct, returning `$2.4` here.

Reply:
Hey @TylerAPfledderer I think that the problem is that `avgBlobFee` is in wei, not gwei. That is why we get it in billions. So, if we do

```suggestion
  const ethPrice = useEthPrice()
  const avgBlobFeeUsd = formatSmallUSD(
    // Converting value from wei to USD
    avgBlobFee * 1e-18 * ethPrice,
    localeForNumberFormat
  )
```

we shouldn't need to replace anything.

Reply:
@pettinarip ah, that did it! Pushed the change 😄